### PR TITLE
update plymouthd module

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: plymouthd.if [hunk 1]
```diff
@@ -214,21 +214,21 @@ interface(`plymouthd_manage_lib_files',`
...
+##	Read plymouthd runtime files.
 ## </summary>
 ## <param name="domain">
 ##	<summary>
 ##	Domain allowed access.
 ##	</summary>
 ## </param>
 #
-interface(`plymouthd_read_pid_files',`
+interface(`plymouthd_read_run_files',`
```
Comment (pebenito) [inline plymouthd.if]: Missing compat interface, and should be `plymouthd_read_runtime_files`.

File: plymouthd.if [hunk 2]
```diff
@@ -264,5 +264,13 @@ interface(`plymouthd_admin',`
...
 	files_search_pids($1)
-	admin_pattern($1, plymouthd_var_run_t)
+		admin_pattern($1, plymouthd_runtime_t)
+
+	ifdef(`init_system',`
+		gen_require(`
+			class service { start status stop };
+		')
+
+		allow $1 plymouthd_unit_t:service { start status stop };
```
Comment (pebenito) [inline plymouthd.if]: `init_startstop_service`

Comment (cgzones) [inline plymouthd.if]: Is it reasonable to allow the 4th argument to `init_startstop_service` to be empty?

Comment (pebenito) [inline plymouthd.if]: Not as long as we still have sysvinit distros supported.

**Comment by cgzones** [comment]
close for now

**Pull request closed without merging**